yaks-rails/lib/yaks-rails.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails'
+require 'yaks-html'
+
+module Rails
+  module Yaks
```

Comment:
Better switch this around. `Rails` is not our namespace, we should not add to it. so `Yaks::Rails` instead of `Rails::Yaks`